### DISCUSSION TITLE
Error while installing fasjet

### DISCUSSION THREAD

**Discussion by sandeepts224** [MadAnalysis/madanalysis5#8] [Q&A]
when I try to install fastjet by typing  "install fastjet" following error came in.
What should I do?

MG5_aMC>install fastjet
syntax: install Delphes|MadAnalysis4|ExRootAnalysis|update|Golem95|QCDLoop|maddm|maddump|looptools|MadSTR|pythia8|zlib|boost|lhapdf6|lhapdf5|collier|hepmc|mg5amc_py8_interface|ninja|oneloop|MadAnalysis5
-- Download the last version of the program and install it
   locally in the current MadGraph5_aMC@NLO version. In order to have
   a successful installation, you will need to have an up-to-date
   F77 and/or C and Root compiler.
 
   When installing any of the following programs:
     pythia8, zlib, boost, lhapdf6, lhapdf5, collier, hepmc, mg5amc_py8_interface, ninja, oneloop, MadAnalysis5
   The following options are available:
     --force        Overwrite without asking any existing installation.
     --keep_source  Keep a local copy of the sources of the tools MG5_aMC installed from.
 
   "install update"
   check if your MG5 installation is the latest one.
   If not it load the difference between your current version and the latest one,
   and apply it to the code. Two options are available for this command:
     -f: didn't ask for confirmation if it founds an update.
     --timeout=: Change the maximum time allowed to reach the server.
Command "install fastjet" interrupted with error:
InvalidCmd : Not recognize program fastjet 
MG5_aMC>install fastjet
syntax: install Delphes|MadAnalysis4|ExRootAnalysis|update|Golem95|QCDLoop|maddm|maddump|looptools|MadSTR|pythia8|zlib|boost|lhapdf6|lhapdf5|collier|hepmc|mg5amc_py8_interface|ninja|oneloop|MadAnalysis5
-- Download the last version of the program and install it
   locally in the current MadGraph5_aMC@NLO version. In order to have
   a successful installation, you will need to have an up-to-date
   F77 and/or C and Root compiler.
 
   When installing any of the following programs:
     pythia8, zlib, boost, lhapdf6, lhapdf5, collier, hepmc, mg5amc_py8_interface, ninja, oneloop, MadAnalysis5
   The following options are available:
     --force        Overwrite without asking any existing installation.
     --keep_source  Keep a local copy of the sources of the tools MG5_aMC installed from.
 
   "install update"
   check if your MG5 installation is the latest one.
   If not it load the difference between your current version and the latest one,
   and apply it to the code. Two options are available for this command:
     -f: didn't ask for confirmation if it founds an update.
     --timeout=: Change the maximum time allowed to reach the server.
Command "install fastjet" interrupted with error:
InvalidCmd : Not recognize program fastjet

**Comment by jackaraz** [comment]
Hi, @sandeepts224; for bug reports, please use the relevant [issues](https://github.com/MadAnalysis/madanalysis5/issues) section. There is a reason why discussions are created separately because we need to collect relevant information to solve your problem. Also, this is not a MadAnalysis issue you are using MadGraph please refer to the developers of the MadGraph. MadGraph and MadAnalysis are not the same software...
